apidoc/Titanium/UI/iOS/iOS.yml

```diff
@@ -27,13 +27,16 @@ methods:
     summary: |
         Get image from SF Symbols provided by Apple.
     description: |
-        More details on SF Symbols can be found on https://developer.apple.com/design/human-interface-guidelines/sf-symbols/overview.
-        One can download SF Symbols app from https://developer.apple.com/design/downloads/SF-Symbols.dmg and 
-        use it to lookup the inbuilt SF Symbol name.
+        More details on SF Symbols can be found [here](https://developer.apple.com/design/human-interface-guidelines/sf-symbols/overview).
+        One can download SF Symbols app from [here](https://developer.apple.com/design/downloads/SF-Symbols.dmg) and
+        use it to lookup the built-in SF Symbol name.
     parameters:
       - name: name
         summary: Name of SF Symbol.
         type: String
+      - name: parameters
```

Comment:
likely needs a `since` value of `10.0.0` and also possibly an `osversion` value?

Reply:
oh and `optional: true`

Reply:
Added `since` and `optional` keys. You mean `osver`? Thats already defined by the method docs (iOS 13+) and the "new" API was also added in iOS 13 already.

Reply:
> [ERROR] Titanium.UI.iOS: found 1 error(s), 0 warning(s)!
	methods[systemImage].parameters[parameters].since - Invalid key(s) in Titanium.UI.iOS: since

Nearly forgot: The `since` key is not available everywhere!